platform/kube/controller.go

```diff
@@ -363,6 +363,26 @@ const (
 // "spiffe://cluster.local/ns/foo/sa/bar".
 func (c *Controller) GetIstioServiceAccounts(hostname string, ports []string) []string {
 	saSet := make(map[string]bool)
+
+	// Get the service accounts allowed for the service running on VMs, specified by the Service
```

Comment:
Never mind - I think I see why we may need multiple service account in some cases, but we must have sane defaults for common cases, to keep simple things simple. 

Most common case would be for a service having a single service account, and that can be expressed with the (standard) k8s syntax. Maybe this case will be sufficient for 0.2 ?